arangod/RocksDBEngine/RocksDBRestReplicationHandler.cpp

```diff
@@ -813,3 +838,12 @@ void RocksDBRestReplicationHandler::handleCommandDump() {
     }
   }
 }
+
+#ifdef ARANGODB_ENABLE_FAILURE_TESTS
+/// @brief patch quickKeysNumDocsLimit for testing
+void RocksDBRestReplicationHandler::adjustquickKeysNumDocsLimit() {
+  TRI_IF_FAILURE("RocksDBRestReplicationHandler::quickKeysNumDocsLimit100") {
+    handler._quickKeysNumDocsLimit = 100;
```

Comment:
I wonder how this will compile. Shouldn't this just be:
```suggestion
    _quickKeysNumDocsLimit = 100;
```